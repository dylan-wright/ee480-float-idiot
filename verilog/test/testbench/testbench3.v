op = `OPmulf;
x = 3;
y = 3;
#1 $display("%h", z);
x = 20;
y = 10;
#1 $display("%h", z);
