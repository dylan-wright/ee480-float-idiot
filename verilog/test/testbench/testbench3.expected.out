4110
4348
